packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -323,22 +387,26 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     return this?.state?.citations;
   }
 
-  get hasCitations() {
-    return !!this.citations.length;
+  get shouldDisplayCitations() {
+    return !!this.citations.length && !this.isAnswerCollapsed;
   }
 
   get isStreaming() {
     return this?.state?.isStreaming;
   }
 
   get shouldDisplayActions() {
```

Comment:
I would add a comment here explaining the logic of the binary operation, otherwise it can be hard to grasp at first glance.

Reply:
Really, to me the comment 
`// Only display actions when the answer is visible, not while streaming, and not collapsed.` is longer to read than the true/false :D